bin/dist_get

```diff
@@ -13,6 +13,26 @@ check_writeable() {
 	printf "" > "$1" && rm "$1"
 }
 
+try_download() {
+	url="$1"
+	output="$2"
+	command="$3"
+	util_name=$(set -- $3; echo $1)
```

Comment:
This sets arguments within this shell substitution from `$3`, allowing us to get the first word of the command (wget, curl, http) with `$1`.
Arguments outside this shell substitution are not clobbered.